it.tests/pom.xml

```diff
@@ -35,6 +35,16 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>io.github.valtech-ch</groupId>
+            <artifactId>saas-aem.api</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.github.valtech-ch</groupId>
+            <artifactId>saas-aem.core</artifactId>
```

Comment:
why do we need core dependency?

Reply:
to be able to use the constant SearchTabModelImpl.SEARCH_TERM

Reply:
OK, please move this to the api. It's clearly something the implementer needs to know